vendor/assets/javascripts/i18n-patches.js

```diff
@@ -13,7 +13,7 @@
       return oldI18ntoHumanSize.apply(this, [number, options]);
     };
 
-    if (Ember.typeOf(String.prototype.w) === "function") {
+    if ("w" in String.prototype) {
```

Comment:
Is that syntax supported in all the browsers we support?

Reply:
Yes it is